crates/ruma-client-api/src/room/get_event_by_timestamp.rs

```diff
@@ -0,0 +1,65 @@
+//! `GET /_matrix/client/*/rooms/{roomId}/timestamp_to_event`
+//!
+//! Get the ID of the event closest to the given timestamp.
+
+pub mod unstable {
+    //! `/unstable/` ([spec])
+    //!
+    //! [spec]: https://github.com/matrix-org/matrix-spec-proposals/pull/3030
```

Comment:
Should the link say `MSC` instead of `spec`?

Reply:
I don't know, I followed the lead of other unstable endpoints that use `spec`.

Reply:
Let's do "MSC3030", we have a bunch of links like that in other places.